### PR TITLE
Fixes singuloth not disappearing after a set amount of time.

### LINKED ISSUE #37677: Orion Trail Realism Mode Singularity does not wink out
Issue reported from  Round ID: 87468 (/tg/Station Sybil [ENGLISH] [US-EAST] [100% LAG FREE])

Reporting client version: 512

As stated in the title, the orion trail generated singularity does not disappear after a short period of time, and instead remains indefinitely. An attempt to BoH the stage 1 singularity failed somehow, resulting in an explosion, but no deletion of the singularity.

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Dax Dupont
fix: The singularity spawned by the arcade orion tale meme will disappear as intended.
/:cl:

[why]:  Fixes #37677

### CODE REVIEW

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -614,7 +614,7 @@
 						L.Stun(200, ignore_canstun = TRUE) //you can't run :^)
 					var/S = new /obj/singularity/academy(usr.loc)
 					addtimer(CALLBACK(src, /atom/movable/proc/say, "[S] winks out, just as suddenly as it appeared."), 50)
```
Comment (AlexTheSysop42) [inline code/game/machinery/computer/arcade.dm]: It seems like this should call on the singularity as well

Comment (AlexTheSysop42) [inline code/game/machinery/computer/arcade.dm]: It seems like this should call on the singulo as well

Comment (DaxDupont) [reply code/game/machinery/computer/arcade.dm]: I think the text is supposed to come from the arcade machine.

**Comment by Cyberboss** [comment]
Jesus fuck arcade code is horrid

File: code/game/machinery/computer/arcade.dm (hunk 1 above)
Comment (DaxDupont) [reply code/game/machinery/computer/arcade.dm]: No, it spawns it where the player is.
It's the academy singuloth which doesn't move or grow.

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**